commit 8240a62dc3fd1674719d74ff21f227ed3c3b1f01 (refs/changes/43/4943/1)
Author: Nahuel Greco <ngreco@gmail.com>
Date:   2018-10-11T16:09:09-07:00 (12 months ago)
    
    joyent/node-sshpk#50 Support PKCS#5 AES-256-CBC encrypted private keys

diff --git a/lib/utils.js b/lib/utils.js
index 4dcaf9c..14fee68 100644
--- a/lib/utils.js
+++ b/lib/utils.js
@@ -87,7 +87,8 @@ function assertCompatible(obj, klass, needVer, name) {
 
 var CIPHER_LEN = {
 	'des-ede3-cbc': { key: 7, iv: 8 },
-	'aes-128-cbc': { key: 16, iv: 16 }
+	'aes-128-cbc': { key: 16, iv: 16 },
+	'aes-256-cbc': { key: 32, iv: 16 }
 };
 var PKCS5_SALT_LEN = 8;
 
